source/guides/network/supernode-enrollment.rst

```diff
@@ -0,0 +1,17 @@
+##################################
+Enrolling to the Supernode program
+##################################
+
+**********************
+Using symbol-bootstrap
+**********************
```

Comment:
You can base this guide on this procedure 

https://github.com/nemtech/symbol-bootstrap/pull/92

Reply:
Yeah, that's the plan!